tests/dummy/config/environment.js

```diff
@@ -10,7 +10,8 @@ module.exports = function(environment) {
     locationType: 'router-scroll',
     historySupportMiddleware: true,
     routerScroll: {
-      scrollElement: '#docs-viewer__scroll-body'
+      //scrollElement: '#docs-viewer__scroll-body'
+      scrollElement: 'window'
```

Comment:
There's a bug in router-scroll where it doesn't work with scrollElement+fastboot. I'm going to fix that, but right now this is the best solution.